pxtextension.ts

```diff
@@ -54,14 +63,21 @@ class MlEvent {
   }
 
   //% blockId=mlrunner_is_ml_event
-  //% block="is %this action"
+  //% block="is $this action"
   isEvent(): boolean {
     if (!mlrunner.isRunning()) {
       mlrunner.startRunning();
       return false;
     }
     return this.eventValue == mlrunner.currentActionId();
   }
+
+  //% blockId=mlrunner_on_ml_event_stop
+  //% block="on $this stop $duration"
+  //% draggableParameters="reporter"
+  onStop(body: (duration: number) => void): void {
+    this.onStopHandler = body;
+  }
```

Comment:
In the blows view, this lets you drag a `[duration]` red block and drop it in a block input, outside of the "onStop" event handler.
Do we know how the radio blocks limit this?

Is that something we are still looking into and out-of-scope for this PR? just want to make sure we are tracking it somehow, as it's really useful.

Reply:
I'm wondering if this is a bug when testing an extension, because I've found that I can do exactly that with the radio blocks in this mode: 
![image](https://github.com/user-attachments/assets/df418416-42bc-4e91-b003-63ee9c2f4bc2)

Definitely something to test once we merge and tag a new version.

Reply:
Oh yeah, you are right, that's really weird.
Can the custom MakeCode deployment we have be changed in the configuration to not use the editor extension? (to be able to discard the difference being between or deployment or the extension itself)

Reply:
It does seem to just be a bug in our MakeCode deployment. If you save a hex and drop it into the official version of MakeCode, it works as expected, in the same way as the radio blocks.

Reply:
Ah, that's great to know, thanks Rob!